src/main/java/io/ebeaninternal/server/core/PersistRequestBean.java

```diff
@@ -918,7 +928,7 @@ private void changeLog() {
    */
   @Override
   public void postExecute() {
-
+    postExecute = true;
```

Comment:
Make postExecute as being executed.  We need postExecute and complete to be executed to put the L2 cache notification messages in